### DISCUSSION TITLE
Ordering events by Post Date in CP is not correct

### DISCUSSION THREAD

**Discussion by simonkuran** [solspace/craft-calendar#229] [Feature Requests]
**Describe the bug or issue you're experiencing**
When sorting events by Post Date in the CP the order of events is not correct.

**Steps to reproduce**

1. In the CP go to Calendar > Events
2. Sort list by Post Date (newest first is easiest to see this bug)

**Expected behavior**
The events are in the correct order when sorting.

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 4.4.2
- Calendar Version: 4.0.7
- Calendar Edition: Pro
- Fresh Install or Upgrade: Has been updated, but initial Calendar install was only v4.0.0

**Additional context**
The red text is what the Post Date actually is (as listed on the sidebar of each entry in the CP). Note the date of the first one and the times of the events on Monday.

<img width="254" alt="calendar-post-date-sorting" src="https://user-images.githubusercontent.com/41410727/234517471-ed6c2e20-925e-43fd-b073-6fd8cc844cc5.png">

**Comment by kjmartens** [comment]
Hi @simonkuran,

Sorry for the trouble and the delay. Unfortunately this is an issue that we are unable to resolve at this time. It's on our list to revisit in the future, however. The problem is that Calendar takes on some custom approaches to display its events in the CP. This is due to performance reasons and because Calendar needs to artificially inflate event results if an event recurs.

**Reply by jowhannez** [reply]
Any idea on a time frame for when a solution will be possible? I have some clients that are specifically asking for this functionality

**Reply by kjmartens** [reply]
I don't have any updates on this yet, but we hope to look into it again soon and see what our options are.
